repo/scaffold/application.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+from ramble.appkit import *
+
+
+class Scaffold(ExecutableApplication):
+    """Scaffold benchmark"""
+
+    name = "scaffold"
+
+    tags = ["python"]
+
+    os.system("ml load rocm/6.2.1 rocmcc/6.2.1-cce-18.0.1a-magic")
+    software_spec("scaffold", None)
+
+    executable(
+        "modules",
+        "export LD_LIBRARY_PATH=/opt/cray/pe/cce/18.0.1/cce/x86_64/lib:$LD_LIBRARY_PATH",
+    )
```

Comment:
Is there a better way to load these modules while using pip package manager (not using spack or environment-modules here)?

Reply:
We determined the elegant solution to this would be to create a `environment-modules-pip` package manager in ramble.